core/templates/dev/head/pages/exploration_editor/translation_tab/translator_overview_directive.html

```diff
@@ -14,7 +14,7 @@
     <div class="progress oppia-translation-progess-bar">
       <div class="progress-bar progress-bar-success" role="progressbar" ng-style="getTranslationProgressStyle()"></div>
     </div>
-    <span class="oppia-numerical-progress-status protractor-test-translation-numerical-status">(<[numberOfRequiredAudio - numberOfAudioNotAvailable]>/<[numberOfRequiredAudio]>)</span>
+    <span aria-label="n item translated out of m items" role="status" class="oppia-numerical-progress-status protractor-test-translation-numerical-status">(<[numberOfRequiredAudio - numberOfAudioNotAvailable]>/<[numberOfRequiredAudio]>)</span>
```

Comment:
Can we use the specific value over here?

Reply:
Ya we can.